src/Microsoft.ML.TensorFlow/TensorflowTransform.cs

```diff
@@ -316,7 +667,14 @@ public void Save(ModelSaveContext ctx)
             {
                 ctx.SaveBinaryStream("TFSavedModel", w =>
                 {
-                    string[] modelFilePaths = Directory.GetFiles(_savedModelPath, "*", SearchOption.AllDirectories);
+                    // only these files need to be saved.
+                    string[] modelFilePaths =
+                    {
+                        Path.Combine(_savedModelPath, "saved_model.pb"),
+                        Path.Combine(_savedModelPath, "variables", "variables.data-00000-of-00001"),
+                        Path.Combine(_savedModelPath, "variables", "variables.index"),
```

Comment:
Can we make this strings are const strings variables. You use them in two different places, code wise would be nice to refactor it slightly.
Also is this names are tensor flow specific or it's your deliberate choice?

Reply:
These are strictly specific to TensorFlow. 

---
In reply to: [221771901](https://github.com/dotnet/machinelearning/pull/1063#discussion_r221771901) [](ancestors = 221771901)